committeeoversight/urls.py

```diff
@@ -25,9 +25,9 @@
 from committeeoversightapp.views import pong
 
 urlpatterns = [
-    path(r'^admin/', admin.site.urls),
-    path(r'^pong/$', pong,),
```

Comment:
Got warnings for some lingering Regex I didn't remove when I switched this from `url()`. Fixed in this commit! (https://github.com/datamade/committee-oversight/commit/07d4aea32914092d3b75d1e7f2981eca010c3383)

Reply:
Good catch!